ui/components/src/RefreshIntervalPicker/RefreshIntervalPicker.tsx

```diff
@@ -0,0 +1,50 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Box, FormControl, Stack } from '@mui/material';
+import { DurationString, RelativeTimeRange } from '@perses-dev/core';
+import { TimeOption } from '../model';
+import { RefreshIntervalSelector } from './RefreshIntervalSelector';
+
+interface RefreshIntervalPickerProps {
```

Comment:
Looks like this and `RefreshIntervalSelectorProps` are duplicated. Agreed with Steven that maybe it's better to combine the two components

Reply:
Agreed!
Going to change it